docs/framework/configure-apps/file-schema/section-element.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "&lt;section&gt; element | Microsoft Docs"
+ms.date: "05/01/2017"
+ms.prod: ".net-framework"
+ms.technology: 
+  - "dotnet-clr"
+ms.topic: "article"
+f1_keywords: 
+  - "http://schemas.microsoft.com/.NetConfiguration/v2.0#configuration/configSections/section"
+  - "http://schemas.microsoft.com/.NetConfiguration/v2.0#configuration/configSections/sectionGroup/section"
+dev_langs: 
+  - "VB"
+  - "CSharp"
+  - "C++"
+  - "jsharp"
+helpviewer_keywords: 
+  - "section Element"
+  - "<section> Element"
+ms.assetid: ec7d4110-2403-47ac-8218-499bfe9d5ddb
+author: "guardrex"
+ms.author: "mairaw"
+manager: "wpickett"
+---
+
+# \<section> element
+
+Contains a configuration section declaration.
+
+[**\<configuration>**](~/docs/framework/configure-apps/file-schema/configuration-element.md)   
+&nbsp;&nbsp;[**\<configSections>**](~/docs/framework/configure-apps/file-schema/configsections-element-for-configuration.md)   
+&nbsp;&nbsp;&nbsp;&nbsp;**\<section>**
+
+[**\<configuration>**](~/docs/framework/configure-apps/file-schema/configuration-element.md)   
+&nbsp;&nbsp;[**\<configSections>**](~/docs/framework/configure-apps/file-schema/configsections-element-for-configuration.md)   
+&nbsp;&nbsp;&nbsp;&nbsp;[**\<sectionGroup>**](~/docs/framework/configure-apps/file-schema/sectiongroup-element-for-configsections.md)   
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**\<section>**
+
+## Syntax
+
+```xml
+<section name="section name"
+         type="configuration section handler class, assembly"
+         allowDefinition="Everywhere|MachineOnly|MachineToApplication" 
+         allowLocation="true|false" />
+```
+
+## Required attributes
+
+| Attribute | Description |
+| --------- | ----------- |
+| **name**  | Specifies the name of the configuration section. |
+| **type**  | Specifies the name of the configuration section handler class that reads the section from the configuration file. |
```

Comment:
This needs to have a bit more detail. The **type** value has the syntax "fully-qualified-section-handler-class-name, simple-assembly-name". The simple assembly name is the root filename without the .dll file extension.